src/main/scala/org/broadinstitute/dsde/firecloud/service/LibraryServiceSupport.scala

```diff
@@ -170,26 +171,7 @@ trait LibraryServiceSupport extends DataUseRestrictionSupport with LazyLogging {
     samDAO.listGroups(userInfo) map {FireCloudConfig.ElasticSearch.discoverGroupNames intersect _}
   }
 
-  def updateAccess(docs: LibrarySearchResponse, workspaces: Seq[WorkspaceListResponse]) = {
-
-    val accessMap = workspaces map { workspaceResponse: WorkspaceListResponse =>
-      workspaceResponse.workspace.workspaceId -> workspaceResponse.accessLevel
-    } toMap
-
-    val updatedResults = docs.results.map { document =>
-      val docId = document.asJsObject.fields.get("workspaceId")
-      val newJson = docId match {
-        case Some(id: JsString) => accessMap.get(id.value) match {
-          case Some(accessLevel) => document.asJsObject.fields + ("workspaceAccess" -> JsString(accessLevel.toString))
-          case _ => document.asJsObject.fields + ("workspaceAccess" -> JsString(WorkspaceAccessLevels.NoAccess.toString))
-        }
-        case _ => document.asJsObject.fields
-      }
-      JsObject(newJson)
-    }
-
-    docs.copy(results = updatedResults)
-  }
+  val accessible = Seq(AccessPolicyName("reader"), AccessPolicyName("writer"), AccessPolicyName("owner"))
```

Comment:
is this used anywhere? If so - and depending on its usage pattern - might want to move it up to an object level so we're not constantly creating a new Seq each time it's used.